ui/app/styles/core/buttons.scss

```diff
@@ -337,7 +337,24 @@ a.button.disabled {
     min-width: $spacing-l;
     padding: 0;
   }
+
   &.white-icon svg {
     color: $white;
   }
+
+  &.is-primary {
```

Comment:
Did we add a new `is-primary` class here? Looking at our current buttons stylesheet it looks like we have an `is-primary`, so curious if we want to have two different primary colors.

Reply:
Yes, I added a new `is-primary` class, but this one is a child of the `hds-copy-button` class as opposed to being a child component of the `button` class. This way, when I used this class I would do `class="hds-copy-button is-primary` rather than `class="button is-primary hds-copy-button white-icon is-flex-center`. 

This `is-primary` class I created takes a few necessary styles from the `button` class, such as `height` and `min-width`, and the rest of the styles are borrowed from the other `is-primary` class. The class I made doesn't have anything like disabled, hover, active, etc since I didn't want to add too much extra styling to the HDS copy button.

Anyways, that was my thinking in creating a new `is-primary` class as a child of `hds-copy-button` class. I wanted to use less class names and avoid using the `button` class and its children for any HDS copy buttons. Please let me know if I should change this! This was just my intuition for how to do the styling, but I am not sure if there are better options (such as using the `button is-primary` classes). I'm also not sure if I should add the styling for hover, disabled, etc to the new `is-primary` class, if I keep it.

Reply:
(I assume all this styling will change later, too, to better match the HDS designs. For now I'm trying to use the HDS copy button but match how the structure copy button looked)

Reply:
That makes sense to me - Thanks for clarifying Malina! 😊 I was curious since having two `is-primary` classes that essentially have very similar styles seemed like we could reuse the nested `is-primary` from the button class. However, I just pulled down your branch and saw you have a specific section for `hds-copy-button`. As you said, the styling will likely be removed when the copy buttons get refactored in the future.

Reply:
Sounds good! 😄